doc/script_commands.txt

```diff
@@ -1150,6 +1150,20 @@ will terminate.
 
 ---------------------------------------
 
+*clear;
+
+This command will clear the dialog text and continues the script without player interaction.
+
+	mes"This how clear command work .";
+	sleep2 3000;
+	clear;	//this will clear what in the dialog and move to the next one.
+	mes"i will show you again .";
+	sleep2 3000;
+	clear;
+	mes"see you latter my friend";
```

Comment:
Missing a blank between mes and the parameter in all mes calls.

Reply:
@aleos89 could you do an English cleanup?

Reply:
```
*clear;

This command will clear the dialog text and continue the script without player interaction.

Example:
	mes "This is how the 'clear' script command works.";
	sleep2 3000;
	clear; // This will clear the dialog and continue to the next one.
	mes "I will show you again.";
	sleep2 3000;
	clear;
	mes "Bye!";
	close;

---------------------------------------
```